class RandomNumberException implements Exception {
  final String? title;
  final String? message;
  const RandomNumberException({this.title, this.message});
}
